go/channelling/roomworker.go

```diff
@@ -290,6 +271,11 @@ func (r *roomWorker) Leave(sessionID string) {
 		r.mutex.Lock()
 		if _, ok := r.users[sessionID]; ok {
 			delete(r.users, sessionID)
+			if len(r.users) == 0 {
+				r.mutex.Unlock()
+				r.expired <- true
```

Comment:
> I think only this change is required to immediately wake the room worker when the last user leaves. 

Do we really want that. It might be good to have a grace period before a room is killed.
Imagine having locked a room and reloading the page: The lock would be removed immediately.